wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -54,7 +54,7 @@ All rules are exactly that: *rules*. They are *not* guidelines and may *not* be
 - **The `Auto` mod must be able to achieve the full score (1,000,000) in all difficulties.**
 - **Two notes cannot be placed in a single column at the same timestamp.**
 - **No column can be left empty.** If you want to use fewer keys, change the number of keys in the Song Setup.
-- **There must not be more than 6 notes pressed at any given moment.** Most USB-connected keyboards cannot handle 7 simultaneous inputs at a time. Note that this rule does not apply to ends of long notes, as they are released, not pressed.
+- **There should not be more than 6 notes pressed at any given moment in Insane or lower difficulties.** Using more than 6 notes must also follow a reasonable spread to the next lower difficulty. This rule does not apply to ends of long notes, as they are released, not pressed. 
```

Comment:
"should" should still be "must" as it remains as a rule for insane and lower diffs

Reply:
or move to Guidelines if that wasn't the intention^

also remove the trailing space

Reply:
Applied. Sorry that it took so long. I am too scared to break something unintentionally lmao